### PR TITLE
Сhange url when form are submitted and check urk when page are loaded

### PR DESCRIPTION
**Pull request by ariarzer**

### CODE REVIEW

File: client/view/Form/form.js [hunk 1]
```diff
@@ -86,3 +90,23 @@ async function updateStatsView(query) {
...
+function initUrlControl() {
+  let urlParams = new URLSearchParams(window.location.search);
+
+  if (urlParams.get('query')) {
+    submitForm(urlParams.get('query'));
+  }
+}
+
+function changeUrl(query) {
+  window.history.pushState({}, query, `?query=${query}`)
```
Comment (a-chabin) [inline client/view/Form/form.js:111]: > `?query`

it is better for backward capability to do `q` as before
https://github.com/browserslist/browserl.ist/blob/main/views/index.pug#L10

File: client/view/Form/form.js [hunk 2]
```diff
@@ -86,3 +90,23 @@ async function updateStatsView(query) {
...
+}
+
+function initUrlControl() {
+  let urlParams = new URLSearchParams(window.location.search);
+
+  if (urlParams.get('query')) {
+    submitForm(urlParams.get('query'));
+  }
+}
+
```
Comment (ai) [inline client/view/Form/form.js:109]: Do we need to add `popstate` event listener to send AJAX request on Back button click?

Comment (ariarzer) [reply client/view/Form/form.js:109]: I think it will be used in this way vary rarely. But we can add it later. Can i create an issue?

Comment (ai) [reply client/view/Form/form.js:109]: What page will do in Back press right now?

Comment (ariarzer) [reply client/view/Form/form.js:109]: Form result doesn't change. And i told, that very few people will use this feature for it. 
This is more form give a link to the particular query.

Comment (ai) [reply client/view/Form/form.js:109]: > Form result doesn't change

No, this is a bug for me.

We should not break web platform.

File: client/view/Form/form.js [hunk 3]
```diff
@@ -16,8 +16,12 @@ export function initForm() {
     if (!form.checkValidity()) {
       return
     }
+
     let formData = new FormData(form)
     let query = formData.get('query')
+    if(getUrlQuery() !== query) {
```
Comment (ai) [inline client/view/Form/form.js:22]: Violate Prettier formatting rules.

You need to install Prettier to WebStorm or ask to add Prettier to `pre-commit` hook.

**Pull request merged**